### DISCUSSION TITLE
Monthly report including total for all project per client

### DISCUSSION THREAD

**Discussion by FirstAppSol** [kimai/kimai#3855] [General]
Hi! I've been using Kimai (Cloud) for some time, only for tracking my times on different projects with different clients (so I'm the sole user of this tool) - this tool is great, but there is one thing I'm really missing: a total number of hours spend per client. When I use Monthly report tab there hours per project, but what I'm missing is an information on the top what's the total number of hours I worked for this client this month. I don't use Kimai for invoicing purposes. 

Is this functionality already available (free or paid), if so, where do I find it, if not, is it something that is going to be added sometime soon?

Thanks! 

![image](https://user-images.githubusercontent.com/78878133/219949493-bb756979-e61d-472e-8c06-1da0e7531092.png)

**Comment by kevinpapst** [comment]
You can use the Export or Invoice screen.
Simply filter the last month and you will get a "Preview" box with one line per customer for the chosen date-range:
<img width="1262" alt="Bildschirm­foto 2023-02-19 um 14 07 48" src="https://user-images.githubusercontent.com/533162/219950009-b29aa7ff-3241-4606-a20e-ab06015247ba.png">

**Reply by FirstAppSol** [reply]
Hi,

That'd do the trick! Many thanks, another good reason to contribute and get premium subscription :) 

Do what you do, great tool!

**Reply by kevinpapst** [reply]
Thanks for your trust 😄
